keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -3,6 +3,8 @@
 ## Table of Contents
 
 <!-- toc -->
+- [Object Storage Support](#object-storage-support)
+  - [Table of Contents](#table-of-contents)
```

Comment:
@rrati Are you also proposing a new field, BR.parameters? If so, it's missing in the BR spec.

Reply:
No.  B.parameters will contain BR.<Field> name/value.  This isn't a direct map->map copy like with bc.parameters.  It could be a marshaling of the entire BR struct, but most of that data isn't relevant to the B or the sidecar.  The sidecar needs to know the bucket name information, but atm that's about all it needs that isn't already in the B.

Reply:
Actually, correction to this.  b.parameters will be a copy of bc.parameters, and b.protocol.parameters will be a copy of bc.protocol.parameters.  This will be updated in next version of this PR